apps/web/src/locales/en/messages.json

```
@@ -890,6 +893,48 @@
   "fileFormat": {
     "message": "File Format"
   },
+  "FileEncryptedExportWarningDesc" : {
+    "message": "This file export will be password protected and require the file password to decrypt."
+  },
+  "exportPasswordDescription" : {
+    "message": "This password will be used to export and import this file"
+  },
+  "confirmMasterPassword" : {
+    "message": "Confirm Master Password"
+  },
+  "confirmFormat": {
+    "message": "Confirm Format"
+  },
+  "filePassword": {
+    "message": "File Password"
+  },
+  "confirmFilePassword": {
+    "message": "Confirm File Password"
+  },
+  "accountBackupOptionDescription": {
+    "message": "Leverages your Bitwarden account encryption not master password, to protect the export. This export can only be imported into the current account. Use this to create a backup that cannot be used elsewhere."
```

Typo (per the Figma)
```suggestion
    "message": "Leverages your Bitwarden account encryption, not master password, to protect the export. This export can only be imported into the current account. Use this to create a backup that cannot be used elsewhere."
```